### PR TITLE
present rutils and serapeum

### PR DESCRIPTION
**Pull request by vindarel**
a word about them in the cl21 page.

### CODE REVIEW

File: cl21.md [hunk 1]
```diff
@@ -12,6 +12,24 @@ and brings some new welcome features:
...
+- [rutils](https://github.com/vseloved/rutils/blob/master/docs/ann-rutils.md) -
+  a comprehensive and all-encompassing suite of syntactic utilities to
+  support modern day-to-day Common Lisp development. It adds readtable
+  literal syntax for shorter lambdas, hash-tables and vectors
+  (<code>#`…`</code>, `#h` and `#{…}`, `#v`), some generic functions
+  (whose name start with `generic-`),
+  [shortcuts](https://github.com/vseloved/rutils/blob/master/core/abbr.lisp)
+  for standard operators, and many helpful functions.
+- [Serapeum](https://github.com/TBRSS/serapeum) is a set of utilities
+  beyond Alexandria, as a supplement. See its
```
Comment (pnathan) [inline cl21.md]: might want to annotate what it really does?

Comment (vindarel) [reply cl21.md]: allright, done, but I can't do much more, it's a lot of functions and macros often technical.

**Comment by pnathan** [comment]
Nah, that's good. Just the flavor of the library.

**Comment by vindarel** [comment]
We didn't see you for a while, hope you had nice hollydays ! ;)

**Pull request merged**